Extension/src/LanguageServer/cppBuildTaskProvider.ts

```diff
@@ -156,7 +160,7 @@ export class CppBuildTaskProvider implements TaskProvider {
 
         if (!definition) {
             const taskLabel: string = ((appendSourceToName && !compilerPathBase.startsWith(CppBuildTaskProvider.CppBuildSourceStr)) ?
-                CppBuildTaskProvider.CppBuildSourceStr + ": " : "") + compilerPathBase + " build active file";
+                CppBuildTaskProvider.CppBuildSourceStr + ": " : "") + compilerPathBase + localize("build_active_file", " build active file");
             const filePath: string = path.join('${fileDirname}', '${fileBasenameNoExtension}');
```

Comment:
Do you think the translators will understand this? It seems safter to use "build active file" and add the " " before the translated string; otherwise, the translators might not understand and remove the space.

Reply:
You are right